client/app/_components/ui/Form.tsx

```diff
@@ -85,17 +85,22 @@ FormItem.displayName = 'FormItem'
 
 const FormLabel = React.forwardRef<
   React.ElementRef<typeof LabelPrimitive.Root>,
-  React.ComponentPropsWithoutRef<typeof LabelPrimitive.Root>
->(({ className, ...props }, ref) => {
+  React.ComponentPropsWithoutRef<typeof LabelPrimitive.Root> & {
+    isRequired?: boolean
```

Comment:
I added `isRequired` props to show required mark. Please add it to the components where you made whenever you can!